pkg/security/probe/hookpoints.go

```diff
@@ -78,6 +78,14 @@ var allHookPoints = []*HookPoint{
 		}},
 		EventTypes: []eval.EventType{"chown"},
 	},
+	{
+		Name: "mnt_want_write_file_path", // used on old kernels (RHEL 7)
+		KProbes: []*ebpf.KProbe{{
+			EntryFunc: "kprobe/mnt_want_write_file_path",
+		}},
+		EventTypes: []eval.EventType{"chown"},
```

Comment:
If I'm not mistaken we'll need to add `setxattr` and `removexattr` in https://github.com/DataDog/datadog-agent/pull/6314 once this PR is merged.

Reply:
Indeed ! I removed "setxattr" and "removexattr" when cherry-pick this commit to create this PR